config/dev.config

```diff
@@ -10,18 +10,16 @@
     ]},
 
     {kernel, [
-        {logger_level, info},
+        {logger_level, debug},
         {logger, [
             {handler, default, logger_std_h, #{
                 config => #{type => standard_io},
+                filter_default => log,
                 filters => [
-                    % Filter out supervisor progress reports so
-                    % TLS certificates are not swamping the console...
-                    {filter_out_progress, {
-                        fun grisp_connect_logger_bin:filter_out/2,
-                        {supervisor, report_progress}}}
+                    {disable_progress, {fun logger_filters:progress/2, stop}}
```

Comment:
It would be good to keep the comment why we do that here and also add it to `local.config`, `test.config`.